### DISCUSSION TITLE
如何获取到消息的回复文本

### DISCUSSION THREAD

**Discussion by htharoldht** [nonebot/discussions#212] [Polls]
在群聊里，如果有人回复了某条消息，同时触发了某命令，此时我想获取到回复消息的内容，从而形成A: xxx，B: xxx的形式

但是我翻遍了文档，好像没有能够获取到回复消息内容的，只看到了触发回复的内容

**Comment by yanyongyu** [comment]
`event.reply`里

**Comment by htharoldht** [comment]
okk, 谢谢啦
